pandas/io/tests/test_gbq.py

```diff
@@ -296,6 +296,12 @@ def test_download_dataset_larger_than_200k_rows(self):
         df = gbq.read_gbq("SELECT id FROM [publicdata:samples.wikipedia] GROUP EACH BY id ORDER BY id ASC LIMIT 200005", project_id=PROJECT_ID)
         self.assertEqual(len(df.drop_duplicates()), 200005)
 
+    def test_zero_rows(self):
+        df = gbq.read_gbq("SELECT * FROM [publicdata:samples.wikipedia] where timestamp=-9999999", project_id=PROJECT_ID)
```

Comment:
pls add the issue number here

compare the resultant dataframe with a constructed one and use
``assert_frame_equal(result,expected)``